packages/renderer/src/lib/container/ContainerActions.svelte

```diff
@@ -202,6 +209,12 @@ if (dropdownMenu) {
     menu="{dropdownMenu}"
     detailed="{detailed}"
     icon="{faArrowsRotate}" />
+  <ListItemButtonIcon
+    title="Export Container"
```

Comment:
the full length is 'Exports container's filesystem contents as a tar archive and saves it on the local machine'

I don't know if we can add a custom tooltip there ?